framework/resources/Functional/s3/views/view.sql

```diff
@@ -0,0 +1,4 @@
+create or replace view s3.tmp.`gitignore/drill_view` as select * from cp.`employee.json` limit 7;
+--@test
```

Comment:
remove the --@test as this is now no longer needed?

Reply:
Done.